internal/pkg/cli/svc_deploy.go

```diff
@@ -75,6 +75,10 @@ type deploySvcOpts struct {
 	buildRequired     bool
 }
 
+var validPlatforms = []string{
+	fmt.Sprintf(fmtOSArch, exec.LinuxOS, exec.Amd64Arch),
+}
```

Comment:
Should this variable be defined in the `exec` pkg? where the other constants are